etc/kayobe/inventory/group_vars/cis-hardening/cis

```diff
@@ -55,7 +55,8 @@ rhel9cis_no_world_write_adjust: false
 rhel9cis_auditd:
   space_left_action: syslog
   action_mail_acct: root
-  admin_space_left_action: halt
+  admin_space_left_action: syslog
+  max_log_file: 10
```

Comment:
Why are we adding max log file here?

Reply:
> This keyword specifies the maximum file size in megabytes. When this limit is reached, it will trigger a configurable action. The value given must be numeric.

10 MB is quite small.

Reply:
It's just specifying the default
https://github.com/ansible-lockdown/RHEL9-CIS/blob/devel/defaults/main.yml#L798

I can remove it if you want?

Reply:
Let's keep the default for now, consider a separate PR for increasing the size?